### PR TITLE
Add escaping in write()

### PR DESCRIPTION
**Pull request by Puttepingvin**
Multiple colons no longer cut the string short. The first colon can be escaped with a hashtag. 

Escape character can be changed but using backslash causes unexpected behaviour.

### CODE REVIEW

File: build/theater.js [hunk 1]
```diff
@@ -223,10 +223,9 @@
 
         if (typeof scene === "string") {
           var params   = scene.split(":"),
-              hasActor = params.length > 1,
+              hasActor = (params.length > 1 & params[0][params[0].length-1] !== '#'),
```
Comment (Zhouzi) [inline build/theater.js]: The backslash is commonly used to escape special characters, `\:` is more appropriate than `#:`

File: build/theater.js [hunk 2]
```diff
@@ -223,10 +223,9 @@
...
               actor    = hasActor ? params[0].trim() : null,
-              speech   = hasActor ? params[1] : params[0];
-
+              speech   = hasActor ? (params.shift(), params.join(":")) : (params[0] = params[0].replace(/#+$/, ""), params.join(":"));
```
Comment (Zhouzi) [inline build/theater.js]: It tends to get too complex here. Feel free to split it on several lines to make it more readable.

Also, `.replace(/#+$/, "")` will remove all the `#` in `I am your##: father` while it should remove only the preceding `#`. `.replace(/#$/, "")` would do the job. That been said, I'd suggest to join before replacing what's needed (for simplicity sake).

```javascript
params[0] = params[0].join(":").replace(/#:/g, ":");
```

**Comment by Zhouzi** [comment]
Thank your very much for your contribution @Puttepingvin!

Your implementation is simple, efficient and does the job perfectly. I just made some remarks, would you mind taking a look and submit a new pull request?

**Comment by Puttepingvin** [comment]
I made a commit with the changes. Do you want me to create a new pull request aswell and delete this one?

**Comment by Zhouzi** [comment]
Thank you @Puttepingvin :+1:

**Pull request merged**